samples/server/petstore/java-play-framework-controller-only/app/controllers/PetApiController.java

```diff
@@ -17,6 +17,7 @@
 import swagger.SwaggerUtils;
 import com.fasterxml.jackson.core.type.TypeReference;
 
+
```

Comment:
There is a lot of new unnecessary blank line. Can you change the mustache to prevent that?

Reply:
In the best case scenario, your changes would not create anything new in the old samples